tooling/minitest/src/tests/unwinding.rs

```diff
@@ -0,0 +1,204 @@
+use crate::*;
+
+#[test]
+fn abort_in_cleanup() {
+    let mut p = ProgramBuilder::new();
+    let mut f = p.declare_function();
+
+    let cleanup = f.cleanup(|f| {
+        f.abort();
+    });
+
+    f.start_unwind(cleanup);
+    let f = p.finish_function(f);
+    let p = p.finish_program(f);
+    dump_program(p);
+    assert_abort::<BasicMem>(p, "aborted");
+}
+
+#[test]
+fn start_unwind_in_main() {
+    let mut p = ProgramBuilder::new();
+    let mut f = p.declare_function();
+
+    let cleanup = f.cleanup(|f| {
+        f.print(const_int(42));
+        f.exit(); //Call exit() instead of abort, because there is currently no way to access the standard output if the program has aborted.
```

Comment:
```suggestion
        // Call exit() instead of abort, because there is currently no way to access the standard output if the program has aborted.
        f.exit();
```